docusaurus.config.js

```diff
@@ -43,7 +43,10 @@ const config = {
                     { loader: 'babel-loader' },
                     {
                       loader: '@linaria/webpack-loader',
-                      options: { sourceMap: dev },
+                      options: {
+                        plugins: ['react-remove-properties', { properties: ['data-test-id'] }],
```

Comment:
the `data-testid` is still in the exported html, in the preview link. 
please change `data-test-id` to `data-testid`, thanks :pray: